Gemfile

```diff
@@ -40,5 +40,5 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 group :development, :test do
-  gem 'rubocop', '~> 0.33.0', require: false
+  gem 'rubocop', '~> 0.34.0', require: false
```

Comment:
@bf4 @joaomdmoura because rubocop release this version and seems no [break change](https://github.com/bbatsov/rubocop/blob/master/CHANGELOG.md#change-log) between 0.33 and 0.34.

:smile: